### PR TITLE
PUT/Bundle Response Manifest

### PR DESCRIPTION
**Pull request by amarjandu**
… (200), attempting pull request to run testing

[WIP] #1331 

PR to test implementation within Travis CI

### CODE REVIEW

File: dss-api.yml [hunk 1]
```diff
@@ -1375,8 +1375,12 @@ paths:
                 description: Timestamp of bundle creation in RFC3339.
                 type: string
                 format: date-time
+              manifest:
+                type: object
+                description: Manifest data regarding data from the bundle
```
Comment (Bento007) [inline dss-api.yml]: Change to "The manifest stored in the DSS. It lists the files in the bundle, and metadata describing those files."

Comment (Bento007) [reply dss-api.yml]: Address this comment and I'll approve

File: tests/test_bundle.py [hunk 2]
```diff
@@ -726,6 +726,7 @@ def put_bundle(
                 }
             )
             self.assertIn('version', resp_obj.json)
+            self.assertIn('manifest', resp_obj.json)
```
Comment (Bento007) [inline tests/test_bundle.py]: add verification for the following manifest fields `name`,`uuid`,`version`,`indexed`

File: dss-api.yml (hunk 1 above)
Comment (amarjandu) [reply dss-api.yml]: complete

**Comment by amarjandu** [comment]
@Bento007 needs review, tests working, dss_api.yml  descriptions updates.

File: tests/test_bundle.py [hunk 3]
```diff
@@ -419,6 +419,20 @@ def _test_bundle_put(self, replica: Replica, fixtures_bucket: str):
                 bundle_version,
             )
 
+        with self.subTest(f'{replica}:  manifest response testing'):
```
Comment (Bento007) [inline tests/test_bundle.py]: can you move this subtest after the subtest in line 436

Comment (amarjandu) [reply tests/test_bundle.py]: Complete, awaiting test results.

**Pull request merged**